src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -353,4 +359,28 @@ private function computeErrorsCount(array $containerDeprecationLogs): array
 
         return $count;
     }
+
+    private function extractDeprecationChannel(\Throwable $throwable = null): ?string
+    {
+        if (null === $throwable || !$this->projectDir) {
+            return null;
+        }
+
+        $appPrefix = $this->projectDir.'/src/';
+        $vendorPrefix = $this->projectDir.'/vendor/';
+
+        // Composer packages
```

Comment:
maybe this comment and the one below should be set just after the hardcoded `/vendor/` & `/src/`